handler.go

```diff
@@ -49,6 +52,7 @@ func (handler *Handler) OnHealth(w http.ResponseWriter, r *http.Request) {
 //OnMajor is a handler for bumping the major part for a given project
 func (handler *Handler) OnMajor(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
+	numberOfDeployments.With(prometheus.Labels{"project": vars["project"]}).Inc()
```

Comment:
does it make sense to inc the counter after a successful bump?

Reply:
hm, 
my thinking was:
we will always bump the version when we deploy (at least in our case). 
perhaps we should rename the metric in "bumps" instead of "deployment"?

Reply:
and fyi: id like to increment a metric to make clean prom-grafana-annotations possible and to get some
metrics about deployments/day etc

Reply:
from the vbump view it make sense to rename it to bumps

Reply:
the problem with this is, if you had 2 failures on bumps and one successful bump, your metric count is 3, but you bump at least one version.

Reply:
ah, now i get your concern. 
agreed!
i changed the location to after errorcheck.